docs/source/spec/core.rst

```diff
@@ -3304,6 +3304,52 @@ in Java).
             message: String,
         }
 
+.. _streaming-trait:
+
+``streaming`` trait
+-------------------
+
+Summary
+    Indicates that the the data stored in the shape is very large and should
+    not be stored in memory, or that the size of the data stored in the shape
+    is unknown at the start of a request.
+Trait selector::
+    ``:each(operation -[input, output]-> structure > :test(member > blob), :test(member:of(structure[trait|error]) > blob))``
```

Comment:
Can you provide a comment below the expression to explain the expression? Lots of other traits do this when the selector is complex.

And one other thing... let's remove the ability to use streaming on errors. It's not used in any AWS services today, and removing this ability would allow us to improve polymorphic type serialization so that we don't need the `x-amz-error-target` header anymore. We can always add this back later if we really want to support it.